scripts/zones/Buburimu_Peninsula/Zone.lua

```diff
@@ -27,6 +28,10 @@ zone_object.onInitialize = function(zone)
     xi.helm.initZone(zone, xi.helm.type.LOGGING)
 end
 
+zone_object.onZoneTick = function(zone)
+    xi.garrison.buildNpcTable(zone)
+end
+
```

Comment:
This looks very suspicious.

Should you be building the table on every tick?

If it's a thing that needs to be built once, it can go in zone init, or whatever.

If it's something that _should_ happen every tick, this should probably indicate that it's only for when garrison is active

```
if xi.garrison.isGarrisonActive(zone) then
    xi.garrison.tickGarrison(zone)
end
```

etc.

Reply:
possibly not every tick its there for testing for now.. needs to be moved to something like on conquest update and need to run a check before start to possibly build it as well